### ISSUE TITLE
Can't boot iso from link given.

### ISSUE THREAD

**Issue by streetwolf** [ventoy/Ventoy#1034]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.48

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

GPT

### Disk Capacity

512GB

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

https://1drv.ms/u/s!AtusZi2hwX-rhJtqF1pUnPdacVj6-Q?e=uZiWzs

### What happened?

Boot hangs with no error messages.

**Comment by ventoy** [comment]
1. Please make a video about the problem.
2. Try press F7 to switch to TEXT mode and then select the ISO file to boot
3. Try press `Ctrl+r` to enter grub2 mode and then select the ISO file to boot

**Comment by streetwolf** [comment]
Option #3 works!  The only gotcha is when I exit the application and I select to restart my computer I hang with some sort of watchdog error.  Not a big deal as I just have to power off and then on again to boot.


……..…….
Regards,
Gary

**Comment by ventoy** [comment]
`Boot hangs with no error messages.`
Did you see the boot menu after you select the ISO file to boot?
Or after you select the ISO file and press Enter to boot, then it hang without error message?
Did it still show the Ventoy screen?
A video is better.

**Comment by streetwolf** [comment]
When I boot into the Ventoy screen I press Ctrl+r which gets me into GRUB mode.  I then select the ISO from the list.  The ISO immediately boots up into my application.  I am able to perform all functions.  As I mentioned the only issue is when I exit out of my app and it asks if I want to restart my machine it appears to hang.  This is not a problem as I just power off then power on my machine.  I am happy the way it is working.

**Comment by ventoy** [comment]
If you don't use grub2 mode to boot.

`Boot hangs with no error messages.`
Did you see the boot menu after you select the ISO file to boot?
Or after you select the ISO file and press Enter to boot, then it hang without error message?
Did it still show the Ventoy screen?
A video is better.

**Comment by streetwolf** [comment]
Did you see the boot menu after you select the ISO file to boot?
  Not sure what boot menu you mean.  To me the boot menu is the Ventoy menu that has all my iso’s on it.  What boot menu are you referring to?

Were you able to download the ISO I gave the link to?  If so you can boot it up and see what’s happening. If you are having problems downloading it from my OneDrive maybe I can upload it to a site of your choosing.



……..…….
Regards,
Gary

**Comment by ventoy** [comment]
I can boot the ISO file in legacy BIOS mode.
After I select the ISO file and press Enter to boot.
I get this boot menu:

![image](https://user-images.githubusercontent.com/59477474/127882799-53a8d320-3855-4620-aba6-35a906ee1307.png)

**Comment by ventoy** [comment]
Can you see this boot menu in Legacy BIOS mode? (don't use grub2 mode)

**Comment by ventoy** [comment]
There is a problem when I boot in UEFI mode in VMWare but I have fixed it : https://github.com/ventoy/Ventoy/actions/runs/1090665431

**Comment by streetwolf** [comment]
I also get to the same point as you but it displays different than what you show.  When the app starts up there is a progress bar that goes about halfway and then the screen shows a bunch of messages below.

***@***.***
……..…….
Regards,
Gary

**Comment by streetwolf** [comment]
I used the updated version of Ventoy given in the link you provided.  Here’s my results.


  1.  I was able to boot and  StorageCraft’s menu displayed. There was a countdown and the program started.
  2.  I saw the progress bar for a split second.
  3.  I then got the following messages

***@***.***

I pressed Control-D and the program started up with no problems.


When I exited out of the program and attempted a restart I got these messages.

***@***.***

……..…….
Regards,
Gary

**Comment by streetwolf** [comment]
Screenshots to go with my previous post.

![IMG_20210802_112928379_HDR](https://user-images.githubusercontent.com/410799/127902171-5ff1a9e3-726a-4fdc-b204-29b8e45cb0c4.jpg)
 
![IMG_20210802_112813808_HDR](https://user-images.githubusercontent.com/410799/127902207-276b2e77-dfa4-46c1-8fbe-9fceb04b0ec6.jpg)

**Comment by ventoy** [comment]
You can use Rufus to write the ISO file directly to the same USB stick and boot.
To see whether these messages also appears

**Comment by streetwolf** [comment]
Another problem popped up.  When I boot into the StorageCraft program using Ventoy whose iso is on a 3.1 external drive an Image Restore gives me an estimated time of a few hours to complete.  It started off pretty fast and then came this major slowdown.  I don't know if a Backup would do the same thing or not.  Doing a Verify Image works quickly.  A Restore from a USB stick takes about a minute or so.

What is the difference between booting from a USB stick created by Rufus and booting from Ventoy as it relates to the StorageCraft program?  Should the two methods be the same or are they different which would account for the slow speed using Ventoy?

**Comment by streetwolf** [comment]
I tracked down the cause of my problems.  I recently enabled Secure Boot in my BIOS in preparation for Windows 11.  My StorageCraft bootable program which is Linux based does not like Secure Boot.  After I disable SB in my BIOS I don't get the message stating I need to press Ctrl-D to continue.  Instead I just get a progress bar that lasted for a few seconds and the program started right up.  When I did a restart I got different messages but the restart took affect automatically.  The only thing I haven't tried yet is a Restore but I bet that will work fine.
